base/markdown/render/plain.jl

```diff
@@ -37,6 +37,45 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, md::Table)
+    col_widths = reduce(max, map(row -> int(map(x -> length(plaininline(x)), row)), md.rows))
```

Comment:
Right idea, but note that `plaininline` is potentially expensive, so calling it three times redundantly isn't optimal. If I were you I'd map the list of cells to a list of rendered cells (via `plaininline`), then work directly with that for the rest of the function.